src/shared/projections.js

```diff
@@ -0,0 +1,50 @@
+import { applyTransform } from 'ol/extent';
+import { get as getProjection, getTransform } from 'ol/proj';
+import { register } from 'ol/proj/proj4';
+import proj4 from 'proj4';
+
+export async function search(query) {
+  const response = await fetch('https://epsg.io/?format=json&q=' + query);
+  const json = await response.json();
+
+  const results = json['results'];
+  if (results && results.length > 0) {
+    for (let i = 0, ii = results.length; i < ii; i++) {
```

Comment:
What happens if multiple results are returned?

Reply:
The first truthy one is returned. I don't know how some results could be falsy...
Actually, this part was taken from [Reprojection with EPSG.io Search](https://openlayers.org/en/latest/examples/reprojection-by-code.html)